moninag/server/views.py

```diff
@@ -109,17 +107,14 @@ def put(self, request, server_id):
 
         server = Server.get_by_id(server_id)
 
-        if not server:
-            # Server does not exist
-            json_response['error'] = 'Server was not updated.'
-            return JsonResponse(json_response, status=404)
-
-        if not request.user.id == server.user.id:
-            # Server does not belong to user
-            return HttpResponse(status=403)
-
-        server.update(**server_dict)
-        return HttpResponse(status=200)
+        if server:
```

Comment:
what's the point of rewriting prev construction, since it was easy to read probably